### ISSUE TITLE
Show pronouns next to usernames in the post header

### ISSUE THREAD

**Issue by RustyLoaf** [Taullo/Ruffy#35]
### Pitch

Similar to the Mastodon app, Megalodon, and Moshidon, show a user's pronouns in the post header if they have them set on their profile in the "Extra fields" section.

### Motivation

It's nice to know how to address people without having to click through to their profile and potentially losing your place in the timeline by doing so.

**Comment by Taullo** [comment]
I think it would be better to add a dedicated "tagline" field, similar to forum software. I don't really like the way the apps do it since it's a bit opaque to the user how it behaves.

**Comment by RustyLoaf** [comment]
How would a tagline field be different than a Pronouns field? Wouldn't it also require using the "Extra fields" section?

**Comment by Taullo** [comment]
Nah, I would keep it separate from the extra fields. It'd probably be similar in function to just having a second username field. I don't really want to use the Extra Fields for it cuz then you're looking through an array and regexing to find the field that says "Pronouns" (idk how that works with other languages either) 

The downside with that would be federation with software that doesn't support it though. I just think the way the apps handle it is hacky. Plus, I think it'd be cool to have a tagline field for any use case if someone didn't want to just use it for pronouns.

**Comment by RustyLoaf** [comment]
Just add an option between showing taglines or showing pronouns in the Interface Settings. Take the KDE route and have infinite options for every use-case imaginable. 🫡
